lib/migration/manager/manager.go

```diff
@@ -75,9 +75,9 @@ func (m *DataMigrator) Register(ver int, target string, plugin Plugin) error {
 			Message: "Empty Plugin object is not allowed",
 		}
 	}
-	if ver > version.MaxPluginVersion {
+	if ver > feature.MaxPluginVersion {
```

Comment:
feels like we could have a common method in feature package for this check as well. 😄